services/dkp-insights/metadata.yaml

```
@@ -0,0 +1,15 @@
+displayName: DKP Insights
+description: DKP Insights is D2iQ’s analytics tool used to ensure clusters are secure and in debugging, monitoring, and anticipating future issues with your cluster.
+category:
+  - observability
+  - monitoring
+certifications:
+  - airgapped
+type: catalog
```

we might want this to be:
```suggestion
type: platform
```